libsquoosh/src/index.js

```diff
@@ -196,9 +196,10 @@ class ImagePool {
   /**
    * Create a new pool.
    * @param {number} [threads] - Number of concurrent image processes to run in the pool. Defaults to the number of CPU cores in the system.
+   * @param {string | null} - Customize the entry filename for squoosh. 
    */
-  constructor(threads) {
-    this.workerPool = new WorkerPool(threads || cpus().length, __filename);
+  constructor(threads, filename) {
+    this.workerPool = new WorkerPool(threads || cpus().length, typeof filename === 'undefined' || filename === null ? __filename : filename);
```

Comment:
I’d prefer a default value for the function parameter than this ternary.

Reply:
I normally use typescript, can you actually use "filename = null" in babel too ?

Reply:
That’s just a normal Javascript feature. So yes.